app/controllers/orders/detail.js

```diff
@@ -37,12 +37,18 @@ export default GoodcityController.extend(SearchMixin, {
       .get("lastObject");
   }),
 
-  highlightSelectedTab: Ember.computed("tabName", function() {
+  highlightSelectedTabSummary: Ember.computed("tabName", function() {
     return (
       ["client_summary", "contact_summary"].indexOf(this.get("tabName")) >= 0
     );
   }),
 
+  highlightSelectedTabGoods: Ember.computed("tabName", function() {
+    return (
+      ["active_items", "requested_items"].indexOf(this.get("tabName")) >= 0
+    );
+  }),
```

Comment:
As far as I know, this fix is for a separate ticket, please create a new branch from master and move this code to that branch @shreyas098 . We should not merge two tickets together while fixing issues or any other thing for that matter.